src/components/popups/profile-settings/EmailForm.tsx

```diff
@@ -55,8 +56,7 @@ export default function EditEmail({ show, onClose }: EditProfileProps): ReactEle
   const onSave = async (form: FormValues) => {
     setLoading(true);
     try {
-      const { email, emailConfirm } = form;
-      if (email !== emailConfirm) return;
```

Comment:
why was this check removed?

Reply:
the comparison is now being handled by the react-hook-form validation on line 112-114